Sources/Protocols/ReuseViewItem/CollectionViewReusableViewItem.swift

```diff
@@ -19,11 +20,20 @@ public enum ReusableViewType {
     }
 }
 
-public protocol CollectionViewReusableViewItem {
+public protocol CollectionViewReusableViewItem: CollectionViewSiblingCellItem {
```

Comment:
Due to using `CollectionViewSiblingCellItem` protocol by reusable view items I suppose we should rename it to `CollectionViewSiblingItem`